src/com/khadri/hibernate/college/client/CollegeMain.java

```diff
@@ -0,0 +1,64 @@
+package com.khadri.hibernate.college.client;
```

Comment:
change package from com.khadri.hibernate.college.client to com.khadri.hibernate.client